cmd/podman/root.go

```diff
@@ -269,6 +269,9 @@ func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 	lFlags.StringVar(&opts.URI, urlFlagName, uri, "URL to access Podman service (CONTAINER_HOST)")
 	_ = cmd.RegisterFlagCompletionFunc(urlFlagName, completion.AutocompleteDefault)
 
+	// Context option added just for compatibily with DockerCLI.
+	lFlags.String("context", "", "Name of the context to use to connect to the daemon (This flag is a NOOP and provided solely for scripting compatibility.)")
```

Comment:
default should be "default"

Reply:
I actually had that originally but was not crazy about it since it showed on the help message.  But I can add it back.

Reply:
I think we should hide the flag and not document it.  It doesn't add value since it only accepts one value.  But I think it adds great value for cases as in the initial issue.

Reply:
SGTM